src/utils/__tests__/changelog.test.ts

```diff
@@ -516,23 +516,43 @@ describe('generateChangesetFromGit', () => {
       ].join('\n'),
     ],
     [
-      'should omit milestone body if it is empty',
+      'should omit milestone body if it is empty or null',
```

Comment:
Suggest: "if it is falsey" and then throw in an undefined case as well, for good measure.

Reply:
`undefined` can never happen tho, at least given our types and how the API behaves. But I guess it makes sense. It just requires me to update the types again which I don't want to.

Reply:
If you saaaaaaaayyyy sooooooooooo ......... ;)